Core/Core_c.php

```diff
@@ -660,3 +660,90 @@ public static function create(object $referent): WeakReference {}
      */
     public function get(): ?object {}
 }
+
+/**
+ * Stringable interface marks classes as available for serialization
+ * in a string.
+ *
+ * @since 8.0.0
```

Comment:
Could you please change version to 8.0 here and below because 0 patch version doesn't make any sense and we've got rid of 0 patch version in stubs recently.

Reply:
https://github.com/JetBrains/phpstorm-stubs/pull/857/commits/d5d6f1ec6b6fdc2f722fd8473d01f0dbbbc2b50c